cves/CVE-2013-1664.yml

```diff
@@ -1,296 +1,314 @@
+    usually infer them from path names, bug report tags, or other key words
+    used. A single source file is not what we mean by a subsystem. In Django,
+    the "Component" field on the bug report is useful. But there may be other
+    subsystems involved.
+
+    Your subsystem name(s) should not have any dots or slashes in them. Only
+    alphanumerics, whitespace, _, - and @.Feel free to add multiple using a YAML
+    array.
+
+    In the answer field, explain where you saw these words.
+    In the name field, a subsystem name (or an array of names)
+
+    e.g. clipboard, model, view, controller, mod_dav, ui, authentication
+  answer: xmlbomb_20130219.patch
+  name: [xml_parsers, NodeFilter, ElementTree, xml_sax_handler, expatreader]
+interesting_commits:
+  question: |
+    Are there any interesting commits between your VCC(s) and fix(es)?
+
+    Write a brief (under 100 words) description of why you think this commit was
+    interesting in light of the lessons learned from this vulnerability. Any
+    emerging themes?
+  commits:
+  - commit:
+    note:
+  - commit:
+    note:
+i18n:
+  question: |
+    Was the feature impacted by this vulnerability about internationalization
+    (i18n)? An internationalization feature is one that enables people from all
+    over the world to use the system. This includes translations, locales,
+    typography, unicode, or various other features.
+
+    Answer should be boolean. Write a note about how you came to the conclusions
+    you did.
+  answer: true
+  note: |
+    In the source code, they add and XML Parser that checks unicode and ASCII
+ipc:
+  question: |
+    Did the feature that this vulnerability affected use inter-process
+    communication? IPC includes OS signals, pipes, stdin/stdout, message
+    passing, and clipboard. Writing to files that another program in this
+    software system reads is another form of IPC.
+
+    Answer should be boolean.
+  answer: false
+  note: |
+    There were no OS/IPC related operations in the code
+lessons:
+  question: |
+    Are there any common lessons we have learned from class that apply to this
+    vulnerability? In other words, could this vulnerability serve as an example
+    of one of those lessons?
+
+    Leave "applies" blank or put false if you did not see that lesson (you do
+    not need to put a reason). Put "true" if you feel the lesson applies and put
+    a quick explanation of how it applies.
+
+    Don't feel the need to claim that ALL of these apply, but it's pretty likely
+    that one or two of them apply.
+
+    If you think of another lesson we covered in class that applies here, feel
+    free to give it a small name and add one in the same format as these.
+  defense_in_depth:
+    applies: false
+    note:
+  least_privilege:
+    applies: false
+    note:
+  frameworks_are_optional:
+    applies: true
+    note: |
+      Frameworks are innate security hasards as they are not written by you.

```

Comment:
There's a grammar error here (hazards). Also, "innate" may not be a good choice of word, maybe "inherent" is better?

Reply:
good catch